packages/auth/src/authClients/nhost.ts

```diff
@@ -0,0 +1,78 @@
+import type { Session, NhostClient, User } from 'nhost-js-sdk'
+
+export type Nhost = NhostClient
+export type NhostUser = User
+
+import { AuthClient } from './'
+export interface AuthClientNhost extends AuthClient {
+  login(options: {
+    email?: string
+    password?: string
+    provider?: string
```

Comment:
What are the valid values for this string?

Reply:
added.

Reply:
@Tobbe added.

Reply:
Sorry, should have been more clear. The jsdoc is great, but I was thinking an actual TS type as well. Ideally something exported from the nhost js sdk, but otherwise we could add `type NhostProvider = 'google' | 'github' | 'facebook' | 'linkedin'` here

Reply:
@Tobbe I will add it directly here for now. Our auth solution (`hasura-backend-plus` and the corresponding SDK) actually support a few more providers but we haven't added them to the web console (where we define callback URLs, etc). As soon as we do that, I can update this to use the type we already export there.